pkg/profiles/template.go

```diff
@@ -73,6 +73,11 @@ spec:
       # successes or failures.
       isFailure: true
 
+    # A route can definte a request timeout.  Any requests to this route that
+    # exceed the timeout will be canceled.  If unspecified, the default timeout
+    # is '1s' (one second).
+    # timeout: 250ms
```

Comment:
i really like the way these templates are documented. a related ergonomics question: should the outputs of `--proto` and `--open-api` flags produce similar output and/or default values?

Reply:
I'm not sure.  The more extensive comments/documentation we have in each of these outputs, the harder it will be to keep them all up to date and in sync.  So far I've been treating --template as the authoritative "documentation by example" and the other outputs as basically generated code.